test/maneuversbuilder.cc

```diff
@@ -1925,6 +1926,7 @@ TEST(Maneuversbuilder, TestCombineRoundaboutManeuvers) {
   expected_maneuver2.set_roundabout_exit_begin_heading(280);
   expected_maneuver2.set_roundabout_length(1.0);
   expected_maneuver2.set_roundabout_exit_length(2.0);
+  expected_maneuver2.set_roundabout_exit_turn_degree(0);
```

Comment:
why is this zero?

Reply:
I just used the same 0 value as the turn_degree but I'll update to use actual values as well as showing difference between collapsing and not collapsing.